### ISSUE TITLE
Not issue, just an observation, not sure if right or wrong.

### ISSUE THREAD

**Issue by Ongjming** [SmartlyDressedGames/Unturned-3.x-Community#2082]
I realised while fiddling with crafting vanilla stuff and how it works that,

1. Structure type items cannot be used as supplies in craft recipe otherwise that recipe doesn't show nor work.
2. Medical type items also cannot be used as supplies to craft other medical items (other than rag/bandage/dressing which have that Craft to options on themselves)
3. Currently the stated Actions only apply to the item that the file is for.

Is there any way to make crafting options for structures and medical items?
Is there any way to define Action beyond the involved file? (Crafting recipes/Blueprints currently can be on a totally different item that defines what some other items can do, Actions, I have no idea if there is a way to make it directed to the items I want to involve rather than the placeholder recipe store object.)

**Comment by SDGNelson** [comment]
Off the top of my head there should be not be any restrictions as in 1/2
3. Currently actions are shown for the item they were defined for, but you can show an action for a blueprint defined on another item by setting the Source, but I don't think that is what you want. Defining actions for items in different files could be a nice feature

**Comment by Ongjming** [comment]
Placeholder post : to update more later when I fiddle with the recipe more, thanks for the reply!

**Comment by SDGNelson** [comment]
I copied your example from edit 2 and it looks like it works properly: https://i.imgur.com/UYt3T98.png

**Comment by Ongjming** [comment]
I tried out another prior working craft recipe:
Type Supply
ID 38574

Blueprints 6

Blueprint_0_Type Tool
Blueprint_0_Supplies 8
Blueprint_0_Supply_0_ID 261
Blueprint_0_Supply_0_Amount 10
Blueprint_0_Supply_1_ID 262
Blueprint_0_Supply_1_Amount 10
Blueprint_0_Supply_2_ID 263
Blueprint_0_Supply_2_Amount 10
Blueprint_0_Supply_3_ID 264
Blueprint_0_Supply_3_Amount 10
Blueprint_0_Supply_4_ID 265
Blueprint_0_Supply_4_Amount 10
Blueprint_0_Supply_5_ID 266
Blueprint_0_Supply_5_Amount 10
Blueprint_0_Supply_6_ID 267
Blueprint_0_Supply_6_Amount 10
Blueprint_0_Supply_7_ID 268
Blueprint_0_Supply_7_Amount 10
Blueprint_0_Tool 76
Blueprint_0_Product 1192
Blueprint_0_Products 3
Blueprint_0_Level 3
Blueprint_0_Skill Craft
Blueprint_0_Build 27

Blueprint_1_Type Tool
Blueprint_1_Supplies 6
Blueprint_1_Supply_0_ID 255
Blueprint_1_Supply_0_Amount 10
Blueprint_1_Supply_1_ID 256
Blueprint_1_Supply_1_Amount 10
Blueprint_1_Supply_2_ID 257
Blueprint_1_Supply_2_Amount 10
Blueprint_1_Supply_3_ID 258
Blueprint_1_Supply_3_Amount 10
Blueprint_1_Supply_4_ID 259
Blueprint_1_Supply_4_Amount 10
Blueprint_1_Supply_5_ID 260
Blueprint_1_Supply_5_Amount 10
Blueprint_1_Tool 76
Blueprint_1_Product 1193
Blueprint_1_Products 2
Blueprint_1_Level 3
Blueprint_1_Skill Craft
Blueprint_1_Build 27

Blueprint_2_Type Tool
Blueprint_2_Supplies 6
Blueprint_2_Supply_0_ID 255
Blueprint_2_Supply_0_Amount 10
Blueprint_2_Supply_1_ID 256
Blueprint_2_Supply_1_Amount 10
Blueprint_2_Supply_2_ID 257
Blueprint_2_Supply_2_Amount 10
Blueprint_2_Supply_3_ID 258
Blueprint_2_Supply_3_Amount 10
Blueprint_2_Supply_4_ID 259
Blueprint_2_Supply_4_Amount 10
Blueprint_2_Supply_5_ID 260
Blueprint_2_Supply_5_Amount 10
Blueprint_2_Tool 76
Blueprint_2_Product 1192
Blueprint_2_Products 2
Blueprint_2_Level 3
Blueprint_2_Skill Craft
Blueprint_2_Build 27

Blueprint_3_Type Tool
Blueprint_3_Supplies 6
Blueprint_3_Supply_0_ID 255
Blueprint_3_Supply_0_Amount 10
Blueprint_3_Supply_1_ID 256
Blueprint_3_Supply_1_Amount 10
Blueprint_3_Supply_2_ID 257
Blueprint_3_Supply_2_Amount 10
Blueprint_3_Supply_3_ID 258
Blueprint_3_Supply_3_Amount 10
Blueprint_3_Supply_4_ID 259
Blueprint_3_Supply_4_Amount 10
Blueprint_3_Supply_5_ID 260
Blueprint_3_Supply_5_Amount 10
Blueprint_3_Tool 76
Blueprint_3_Product 43
Blueprint_3_Products 2
Blueprint_3_Level 3
Blueprint_3_Skill Craft
Blueprint_3_Build 27

Blueprint_4_Type Tool
Blueprint_4_Supplies 8
Blueprint_4_Supply_0_ID 261
Blueprint_4_Supply_0_Amount 10
Blueprint_4_Supply_1_ID 262
Blueprint_4_Supply_1_Amount 10
Blueprint_4_Supply_2_ID 263
Blueprint_4_Supply_2_Amount 10
Blueprint_4_Supply_3_ID 264
Blueprint_4_Supply_3_Amount 10
Blueprint_4_Supply_4_ID 265
Blueprint_4_Supply_4_Amount 10
Blueprint_4_Supply_5_ID 266
Blueprint_4_Supply_5_Amount 10
Blueprint_4_Supply_6_ID 267
Blueprint_4_Supply_6_Amount 10
Blueprint_4_Supply_7_ID 268
Blueprint_4_Supply_7_Amount 10
Blueprint_4_Tool 76
Blueprint_4_Product 43
Blueprint_4_Products 5
Blueprint_4_Level 3
Blueprint_4_Skill Craft
Blueprint_4_Build 27

Blueprint_5_Type Tool
Blueprint_5_Supplies 6
Blueprint_5_Supply_0_ID 255
Blueprint_5_Supply_0_Amount 10
Blueprint_5_Supply_1_ID 256
Blueprint_5_Supply_1_Amount 10
Blueprint_5_Supply_2_ID 257
Blueprint_5_Supply_2_Amount 10
Blueprint_5_Supply_3_ID 258
Blueprint_5_Supply_3_Amount 10
Blueprint_5_Supply_4_ID 259
Blueprint_5_Supply_4_Amount 10
Blueprint_5_Supply_5_ID 260
Blueprint_5_Supply_5_Amount 10
Blueprint_5_Tool 76
Blueprint_5_Product 119
Blueprint_5_Products 2
Blueprint_5_Level 3
Blueprint_5_Skill Craft
Blueprint_5_Build 27

Master_Bundle_Override core.masterbundle
Bundle_Override_Path /Items/Backpacks/Alicepack

Had 10 of each flares in inventory and blowtorch and max skills. The craft option doesn't show up anymore.
https://steamcommunity.com/sharedfiles/filedetails/?id=2208209710

I subscribed to it and tested in Singleplayer Germany.
![image](https://user-images.githubusercontent.com/28900564/99371041-61efb800-28f9-11eb-80c8-09accdd0c49c.png)

**Comment by SDGNelson** [comment]
If you look in the Menu > Workshop > Errors menu perhaps there will be something?

**Comment by Ongjming** [comment]
Sorry for the late reply, I checked in the past and currently, it states there are no major issues with your files.

**Comment by SDGNelson** [comment]
Also are you sure your asset is being loaded? Since it is using the alicepack item you can spawn it in to see if it was loaded.

**Comment by Ongjming** [comment]
I am so so sorry, for the late reply and my blunder.
I realised that for the craft ammo I posted, while subscribed to it, it showed the red cross. I reenabled it and it worked.
For the initial post that I was fiddling with though, I realised that the issue might be me not naming the folder's name the same as the .dat file.
I am gonna try naming them the same and test it with a variety of recipe too currently.

Thanks so much! I learned a lot through this, and hope others reading this can learn from my mistake.
